build/common/installer/scripts/tomlparser.rb

```diff
@@ -148,6 +149,16 @@ def populateSettingValuesFromConfigMap(parsedConfig)
       ConfigParseErrorLogger.logError("Exception while reading config map settings for kube event collection - #{errorStr}, using defaults, please check config map for errors")
     end
 
+    #Get PV kube-system enrichment setting
+    begin
+      if !parsedConfig[:log_collection_settings][:collect_kube_system_pv_metrics].nil? && !parsedConfig[:log_collection_settings][:collect_kube_system_pv_metrics][:enabled].nil?
```

Comment:
We need to introduce a new 'metric_collection_settings' and put this one as the first there to make this cleaner